engine/src/main/java/org/terasology/rendering/nui/layers/ingame/ExtraMenuScreen.java

```diff
@@ -0,0 +1,45 @@
+package org.terasology.rendering.nui.layers.ingame;
+
+import org.terasology.crashreporter.CrashReporter;
+import org.terasology.engine.GameEngine;
+import org.terasology.engine.LoggingContext;
+import org.terasology.engine.Time;
+import org.terasology.engine.modes.StateMainMenu;
+import org.terasology.network.NetworkMode;
+import org.terasology.network.NetworkSystem;
+import org.terasology.registry.CoreRegistry;
+import org.terasology.registry.In;
+import org.terasology.rendering.nui.CoreScreenLayer;
+import org.terasology.rendering.nui.WidgetUtil;
+import org.terasology.telemetry.TelemetryScreen;
+
+/**
```

Comment:
Avoid empty javadoc blocks like this and please remove them if you find them anywhere you're changing anyway :-)

Or better yet: Fill it in! Write a brief description about what the class does.